webpack.config.js

```diff
@@ -91,6 +91,7 @@ module.exports = () => {
             client: {
                 overlay: {
                     errors: true,
+                    runtimeErrors: false,
```

Comment:
why not ?

Reply:
ping?

Reply:
Because `runtimeErrors` include fetching errors, which can happen a lot for example when zooming at zoom level 20, since ortho-images are not available everywhere at this zoom level.

Reply:
turns out its for errors that are displayed by webpack in overlay of the webpage which can be annoying for fetch errors in our context and these errors are also handled by the browser anyway so ok for me.

Reply:
To be more specific, removing this setting causes what we can see on this screenshot when zooming for instance out of France : 
![Capture d’écran du 2023-07-03 10-41-52](https://github.com/iTowns/itowns/assets/73115044/27ed83f7-ae07-44ad-9894-3f2dafcae8bc)